ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -406,6 +431,69 @@ protected LayoutParams generateLayoutParams(final ViewGroup.LayoutParams p) {
         return generateDefaultLayoutParams();
     }
 
+    //region Card Bounce Animation
+
+    /**
+     * This method will either create a new bounce Animation or start the one that is already created.
+     */
+    @UiThread
+    private void animateFirstCardView() {
+        if (mBounceAnimator == null) {
+            for (int i = 0; i < getChildCount(); i++) {
+                final View child = getChildAt(i);
+                final LayoutParams lp = (LayoutParams) child.getLayoutParams();
+                switch (lp.childType) {
+                    case CHILD_TYPE_CARD:
+                        bounceCardAnimation(child);
+                        return; // Stop after first Card View
+                }
+            }
+        } else {
+            mBounceAnimator.start();
+        }
+    }
+
+    /**
+     *  This method will check to see if the Animate Card feature has been detected and start or stop the
+     *  animation if needed.
+     */
+    public void shouldAnimateCard() {
+        mShouldAnimateCard = !mSettings.isFeatureDiscovered(Settings.FEATURE_ANIMATE_CARD);
```

Comment:
👎 checking if you need to animate the card or not should be in the `PageViewHolder`

Reply:
and the `PageViewHolder` should be responsible for updating whether the `PageContentLayout` is animating the first card or not.